src/framework/preferences/js/PrefsEditor.js

```diff
@@ -380,7 +380,7 @@ var fluid_2_0 = fluid_2_0 || {};
 
         var initialModel = that.initialModel,
             userSelections = fluid.copy(that.model),
-            changedSelections = {};
+            changedPrefs = {};
```

Comment:
Whatever changes on other parts of the model will be saved and no special handling is needed for those parts.